docs/resources/server/advancements.md

```diff
@@ -63,8 +63,8 @@ public ExampleTriggerInstance(EntityPredicate.Composite player, ItemPredicate it
 Typically, trigger instances have a static constructor which allow these instances to be easily created for data generation. These static factory methods can also be statically imported instead of the class itself.
 
 ```java
-public static ExampleTriggerInstance instance(EntityPredicate.Builder playerBuilder, ItemPredicate.Builder itemBuilder) {
-  return new ExampleTriggerInstance(EntityPredicate.Composite.wrap(playerBuilder.build()), itemBuilder.build());
+public static ExampleTriggerInstance instance(ContextAwarePredicate player, ItemPredicate item) {
+      return new ExampleTriggerInstance(player, item);
```

Comment:
Too much added intentation: existing was 2 spaces, you added 4 for a total of 6.
```suggestion
    return new ExampleTriggerInstance(player, item);
```